middle_end/closure/closure_middle_end.ml

```diff
@@ -54,5 +54,7 @@ let lambda_to_clambda ~backend ~filename:_ ~prefixname:_ ~ppf_dump
   let clambda_and_constants =
     clambda, [preallocated_block], constants
   in
+  Compiler_hooks.execute Compiler_hooks.Raw_clambda clambda;
+  Compiler_hooks.execute Compiler_hooks.Clambda clambda;
```

Comment:
(This looks a bit confusing; why have two calls with
the very same contents? Or is it because there is
a difference only when the backend is flambda?)

Reply:
Because there's only a difference when the backend is flambda. I can remove the call to [Raw_clambda] if it makes things clearer.